src/Components/Timer.tsx

```diff
@@ -78,14 +64,12 @@ const Timer = () => {
   useEffect(() => {
     if (gameOver) {
       pause();
-      gameAudio.pause();
     }
-  }, [gameOver, soundOn])
+  }, [gameOver])
 
   const toggleTimer = (pauseGame: boolean) => {
     if (pauseGame) {
       pause();
-      gameAudio.pause();
```

Comment:
haha na, we should keep this one Kris.
the one we need to remove is the one in line 56*
```
onExpire: async () => {
    gameDispatch({type: "gameOver"})
    gameAudio.pause(); // <------- This one :D 
    await setDoc(gameState.roomId, {
      gameOver: true,
      gameWon: false,
    })
  } });
```

Reply:
ahhh yep thought so! fixed :)